modules/govuk_env_sync/manifests/task.pp

```diff
@@ -58,6 +58,20 @@
     content => template('govuk_env_sync/govuk_env_sync_job.conf.erb'),
   }
 
+  file { $temppath:
```

Comment:
The indentation is off in these two `file` blocks.

Reply:
good spot done.

Reply:
I do not think the lock file should be "ensured" - it is rather created by the setlock command to indicate that no auto-update/reboot should happen during the operation

Reply:
removed ensure